rest-service/manager_rest/rest/resources_v3.py

```diff
@@ -387,3 +390,31 @@ def delete(self):
         """
         params = self._get_and_validate_params()
         return PermissionsHandler.remove_permissions(params)
+
+
+class FileServerAccess(SecuredResource):
+    @staticmethod
+    def _verify_tenant(uri, tenant_name):
+        if len(uri) < 3:
+            return
+        tenanted_resources = [
+            config.instance.file_server_blueprints_folder,
+            config.instance.file_server_uploaded_blueprints_folder,
+            config.instance.file_server_deployments_folder
+        ]
+        if uri[1] in tenanted_resources and uri[2] != tenant_name:
```

Comment:
maybe comment... that if we have 3 items in url....
resource_type, tenant, resource_name = uri